examples/with-angular-thirdpartyemailpassword/src/app/auth/supertokens.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import * as SuperTokens from "supertokens-auth-react";
+
+import * as ThirdPartyEmailPassword from "supertokens-auth-react/recipe/thirdpartyemailpassword";
+
+import { Github, Google } from "supertokens-auth-react/recipe/thirdpartyemailpassword";
+import Session from "supertokens-auth-react/recipe/session";
+
+SuperTokens.init({
+    appInfo: {
+        appName: "SuperTokens Demo App", // TODO: Your app name
+        apiDomain: "http://localhost:3001", // TODO: Change to your app's API domain
+        websiteDomain: "http://localhost:4200", // TODO: Change to your app's website domain
+    },
+    recipeList: [
+        ThirdPartyEmailPassword.init({
+            signInAndUpFeature: {
+                providers: [Github.init(), Google.init()],
+            },
+            emailVerificationFeature: {
+                mode: "REQUIRED",
+            },
+        }),
+        Session.init(),
+    ],
+});
```

Comment:
You are calling this only in the auth route. What if this route is never loaded? Therefore this will never be called, and so the home route will break.

If this is being called even if the user doesn't navigate to the auth route, it means that you are loading supertokens-auth-react on each page load anyway - which breaks the point of the code splitting.. how is it working?